### PR TITLE
Windows Universal App Support 

### PR DESCRIPTION
**Pull request by Pavel-Durov**
Just copied the windows8 directory and named it as windows + added to plugin.xml wiring to windows/AppVersionProxy.js.

### CODE REVIEW

File: src/windows/AppVersionProxy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+AppVersionProxy = {
+  getVersionNumber: function (successCallback, failCallback, args) {
```
Comment (candidodmv) [inline src/windows/AppVersionProxy.js]: This method not work on Windows Phone 8.1 (NOKIA Lumia 625).
I've changed that works:
```
  getAppName: function (successCallback, failCallback, args) {
      Windows.ApplicationModel.Package.current.installedLocation.getFileAsync("AppxManifest.xml").then(function (file) {
          Windows.Data.Xml.Dom.XmlDocument.loadFromFileAsync(file).then(function (xdoc) {
              var displayName = xdoc.getElementsByTagName("DisplayName");
              if (displayName && displayName.length === 1) {
                  var name = displayName[0].innerText;
                  successCallback(name);
              } else {
                  failCallback({ code: -1, message: "ERR_DISPLAY_NAME_NOT_FOUND" });
              }
          }, failCallback);
      }, failCallback);
  }
```

Comment (filfat) [inline src/windows/AppVersionProxy.js]: @candidodmv this has nothing to do with 8.1, It's for UWP.

**Comment by filfat** [comment]
I can confirm that this works targeted UWP, Tested on:
- Windows 10 (Desktop)
- Windows 10 (Mobile)
- Windows 10 (Hololense, in emulator)
- Windows 10 (Xbox One)

File: src/windows/AppVersionProxy.js (hunk 1 above)
Comment (candidodmv) [inline src/windows/AppVersionProxy.js]: I'm using this project template https://github.com/apache/cordova-windows running on Windows Phone 8.1 Update and the above method `getAppName`, works only with my implementation

**Comment by timothystewart6** [comment]
:+1:  thanks!

**Comment by beat-at-hartwig** [comment]
I would really like to see this PR merged!

**Pull request closed without merging**